docs/porting/target/cmake.md

```diff
@@ -15,53 +15,84 @@ Note: All the CMake variables above are generated by mbed-tools and their values
 
 ## Mbed targets CMake input file structure
 
+We still support mbed-cli 1 and its requirements for the folders naming. To stay backward compatible, we still use the same naming even with CMake.
+
+Each Mbed OS target should be prefixed with `mbed-`. We translate automatically Mbed OS target names from targets.json to CMake targets. A target from targets.json named `CYSBSYSKIT_01` has CMake target `mbed-cysbsyskit-01`.
+
 ### Vendor selection
 
-Vendor selection is handled in [`targets/CMakeLists.txt`](https://github.com/ARMmbed/mbed-os/blob/master/targets/CMakeLists.txt) using a conditional statement to verify that one of the labels used by a given Mbed target matches a given partner directory. The matching directory is added to directories to build.
+Vendor selection is handled in [`targets/CMakeLists.txt`](https://github.com/ARMmbed/mbed-os/blob/master/targets/CMakeLists.txt). The matching directory is added to directories to build.
```

Comment:
```suggestion
Vendor selection is handled in [`targets/CMakeLists.txt`](../../targets/CMakeLists.txt). A subdirectory is added to the build if one of the CMake targets it contains is referenced.
```

Reply:
`../../targets/CMakeLists.txt` is this valid url ?

Reply:
Try you will see. It's a relative link.

Reply:
No it will not. I thought this file was in Mbed OS.

Reply:
I'll keep it as it is.

This is documentation (one of the things we will consolidate one day to keep code and docs in one place).

Reply:
There is a suggestion for the second sentence `A subdirectory is added to the build if one of the CMake targets it contains is referenced.`

Reply:
Fixing now